src/ceph-disk/ceph_disk/main.py

```diff
@@ -4228,19 +4241,29 @@ def get_oneliner(base, name):
 
 
 def get_dev_fs(dev):
-    fscheck, _, _ = command(
-        [
-            'blkid',
-            '-s',
-            'TYPE',
-            dev,
-        ],
-    )
-    if 'TYPE' in fscheck:
-        fstype = fscheck.split()[1].split('"')[1]
-        return fstype
+    if FREEBSD:
+        fstype, _, ret = command(
+            [
+                'fstyp',
+                '-u',
+                dev,
+            ],
+        )
+        if ret:
```

Comment:
> EXIT STATUS
>     The fstyp utility exits 0 on success, and >0 if an error occurs or the filesystem type is not recognized.
see http://www.unix.com/man-page/freebsd/8/fstyp/, 

so should be `if ret == 0`, i guess?